File: Módulo Python/Códigos/6 - Funções/Exercicio.03.py
#3. Faça um algoritmo para calcular o cubo e o quadrado de todos os números pertencentes a um intervalo, incluindo o 
# limite superior e inferior (utilize uma função criada por você que retorne o valor do cubo e do quadrado do número).

numeroInferior = int(input('Digite o número inferior do intervalo: '))
numeroSuperior = int(input('Digite o número superior do intervalo: '))

def Quadrado(numero):
    return numero ** 2

def Cubo(numero):
    return numero ** 3

for numero in range(numeroInferior, numeroSuperior + 1):
    quadrado = Quadrado(numero)
    cubo = Cubo(numero)

    print(f'Número: {numero}, Quadrado: {quadrado}, Cubo: {cubo}')


    

File: Livro - Módulo Java/6 - Funções/Exercicio.03.java
// 3 - Faça um algoritmo para calcular o cubo e o quadrado de todos os números
// pertencentes a um intervalo, incluindo o limite superior e inferior (utilize
// uma função criada por você que retorne o valor do cubo e do quadrado do número). 

import java.util.Scanner;

public class CalculoCuboQuadrado {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite o limite inferior do intervalo:");
        int limiteInferior = scanner.nextInt();

        System.out.println("Digite o limite superior do intervalo:");
        int limiteSuperior = scanner.nextInt();

        System.out.println("Cubo e quadrado dos números no intervalo [" + limiteInferior + ", " + limiteSuperior + "]:");
        for (int i = limiteInferior; i <= limiteSuperior; i++) {
            int quadrado = calcularQuadrado(i);
            int cubo = calcularCubo(i);

            System.out.println("Número: " + i + ", Quadrado: " + quadrado + ", Cubo: " + cubo);
        }

        scanner.close();
    }

    // Função para calcular o quadrado de um número
    public static int calcularQuadrado(int numero) {
        return numero * numero;
    }

    // Função para calcular o cubo de um número
    public static int calcularCubo(int numero) {
        return numero * numero * numero;
    }
}